src/components/Icon/Icon.tsx

```diff
@@ -360,6 +362,8 @@ const Icon = ({ type, color, ...props }: Props) => {
     return <SDeleteIcon src={svgIcons['nav-close']} color={color} {...props} />;
   } else if (type === 'address-book') {
     return <SInlineSVG src={svgIcons[type]} fill="none" {...props} />;
+  } else if (type === 'tx-wallet') {
+    return <SInlineSVG src={svgIcons[type]} fill="none" {...props} />;
```

Comment:
This icon still looks wrong. Also it is the same as `icn-sent`

Reply:
Corrected to use `tx-sent` aka `icn-sent`